test/test_nn.py

```diff
@@ -9477,6 +9444,74 @@ def test_ReflectionPad_empty(self, device):
             inp = torch.randn(3, 0, 10, 10, device=device)
             mod(inp)
 
+    @onlyCUDA
+    @dtypes(*NO_HALF_TENSORTYPES)
```

Comment:
Let's modernize this by specifying the dtypes directly (e.g. (torch.float32, torch.float64))

Reply:
fixed